app/model/commands/PublishAtomCommand.scala

```diff
@@ -141,11 +147,40 @@ case class PublishAtomCommand(id: String, override val stores: DataStores, youTu
     }
   }
 
+  private def parseDescriptionForYoutube(description: String): String = {
+      val html = Jsoup.parse(description)
```

Comment:
Just out of interest, what happens if we send HTML in a description to YouTube? I'm wondering if they clean it or if we just end up with garbage